### DISCUSSION TITLE
Bluefish

### DISCUSSION THREAD

**Discussion by JuliaAgostino** [mayrinck/FOSSonMicrosoftStore#19] [Requests]
Bluefish website: http://bluefish.openoffice.nl/

Bluefish privacy policy: https://www.gnu.org/licenses/gpl-3.0.en.html

Logo: https://upload.wikimedia.org/wikipedia/commons/b/b9/Bluefish-icon.svg

Description: "Bluefish is a free software advanced text editor with a variety of tools for programming and website development. It supports coding languages including HTML, XHTML, CSS, XML, PHP, C, C++, JavaScript, Java, Go, Vala, Ada, D, SQL, Perl, ColdFusion, JSP, Python, Ruby and shell. It is available for many platforms, including Linux, macOS and Windows, and can be used via integration with GNOME or run as a standalone application. Designed as a compromise between plain text editors and full programming IDEs, Bluefish is lightweight, fast and easy to learn, while providing many IDE features. It has been translated into 17 languages."

**Comment by mayrinck** [comment]
Cool! Bluefish is a nice one, the only problem is that the offline installer needs other installations to be fully functional, which means, even if I convert it, it won't work... And, there's no privacy policy, by the way...
